module/icp/algs/skein/skein_block.c

```diff
@@ -240,7 +240,8 @@ Skein_256_Process_Block(Skein_256_Ctxt_t *ctx, const uint8_t *blkPtr,
 
 		ts[1] &= ~SKEIN_T1_FLAG_FIRST;
 	}
-	while (--blkCnt);
+	while (--blkCnt)
```

Comment:
Was this the only way you could get the style warning to disappear? Would moving the while on the same line as the closing brace of its block work?